utilitybelt/utilitybelt.py

```diff
@@ -152,8 +153,22 @@ def ip_to_geo(ipaddress):
 def domain_to_geo(domain):
     """Convert Domain to Geographic Information"""
 
+    # Check if the domain resolves to multiple addresses and warn
+    # the user and suggest a different method
+    _, _, ips = socket.gethostbyname_ex(domain)
+    if len(ips) > 1:
+        warnings.warn("{domain} resolves to multiple ip addresses,".format(domain=domain), RuntimeWarning)
```

Comment:
For context, we only have [one line](https://coveralls.io/builds/2624541/source?filename=utilitybelt%2Futilitybelt.py#L195) that is not getting covered by tests.